app/models/playbook.rb

```diff
@@ -27,6 +27,14 @@ class Playbook < ApplicationRecord
   has_many :moves, dependent: :destroy
   has_many :ratings, dependent: :destroy
   validates :name, presence: true
+  
+  default_scope { unarchived }
```

Comment:
Originally default_scope { where(archived_at: nil) } but then optimized for this.